examples/phi2/README.md

```diff
@@ -2,8 +2,19 @@
 
 ## Get the model
 
+Install the model builder script dependencies
+
+```bash
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+wget https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/models/export.py
```

Comment:
this download step seems unnecessary if we already assume the user has a copy of the repo (the next step directly runs `phi2.py`). perhaps just refer to the script path (e.g., `<repo root>/src/python/models/export.py`)